internal/bminventory/inventory.go

```diff
@@ -2543,6 +2543,17 @@ func (b *bareMetalInventory) updatePlatformParams(params installer.V2UpdateClust
 	if params.ClusterUpdateParams.Platform != nil && common.PlatformTypeValue(params.ClusterUpdateParams.Platform.Type) != "" {
 		updates["platform_type"] = params.ClusterUpdateParams.Platform.Type
 		updates["platform_is_external"] = swag.BoolValue(params.ClusterUpdateParams.Platform.IsExternal)
+		if *params.ClusterUpdateParams.Platform.Type == models.PlatformTypeExternal {
+			if params.ClusterUpdateParams.Platform.External != nil && params.ClusterUpdateParams.Platform.External.PlatformName != nil {
+				updates["platform_external_platform_name"] = params.ClusterUpdateParams.Platform.External.PlatformName
```

Comment:
do we have consts for platform_external_platform_name, platform_external_cloud_controller_manager?

Reply:
no, those are columns names in DB, looks like we never defined any consts for them in the past :thinking: